demo-ng/package.json

```diff
@@ -25,24 +25,24 @@
     "build-ios-bundle": "npm run ns-bundle --ios --build-app"
   },
   "dependencies": {
-    "@angular/animations": "~4.2.5",
-    "@angular/common": "~4.2.5",
-    "@angular/compiler": "~4.2.5",
-    "@angular/core": "~4.2.5",
-    "@angular/forms": "~4.2.5",
-    "@angular/http": "~4.2.5",
-    "@angular/platform-browser": "~4.2.5",
-    "@angular/platform-browser-dynamic": "~4.2.5",
-    "@angular/router": "~4.2.5",
-    "nativescript-angular": "4.2.0",
+    "@angular/animations": "~4.4.1",
+    "@angular/common": "~4.4.1",
+    "@angular/compiler": "~4.4.1",
+    "@angular/core": "~4.4.1",
+    "@angular/forms": "~4.4.1",
+    "@angular/http": "~4.4.1",
+    "@angular/platform-browser": "~4.4.1",
+    "@angular/platform-browser-dynamic": "~4.4.1",
+    "@angular/router": "~4.4.1",
+    "nativescript-angular": "4.4.0",
     "nativescript-drop-down": "file:../bin/dist",
     "reflect-metadata": "~0.1.8",
     "rxjs": "~5.2.0",
-    "tns-core-modules": "3.2.0",
+    "tns-core-modules": "next",
```

Comment:
Can this remain at 3.2.0? Or is it required to have vNext?

Reply:
Very well. Personally, i just prefer having next builds in development so i could tell if things will break before hand.

Reply:
Yes, but `@next` build are very unstable. There could be some TNS problem that would cause the CI builds to fail (like it failed with your first set of changes). So I prefer to stay on the stable versions. Thanks!

Reply:
Yes. True. Wondered why it failed too.